addons/website_sale_stock/static/src/js/variant_mixin.js

```diff
@@ -10,6 +10,7 @@ var xml_load = ajax.loadXML(
     '/website_sale_stock/static/src/xml/website_sale_stock_product_availability.xml',
     QWeb
 );
+require('website_sale.website_sale');
```

Comment:
this line prevent to have this [error](https://runbot.odoo.com/runbot/build/6373013)
It worked by chance before that but adding the websiteSaleCarouselProduct widget revealed the lack